#! /bin/sh /usr/share/dpatch/dpatch-run
## 04_base64.dpatch by  <gregor+debian@comodo.priv.at>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Don't build/test base64.*; causes a build failure on hppa,
## DP: and isn't used anyway.

@DPATCH@
diff -urNad iodine~/src/Makefile iodine/src/Makefile
--- iodine~/src/Makefile	2008-03-01 18:52:44.000000000 +0100
+++ iodine/src/Makefile	2008-03-01 18:53:00.000000000 +0100
@@ -1,5 +1,5 @@
 CC = gcc
-COMMONOBJS = tun.o dns.o read.o encoding.o login.o base32.o base64.o md5.o common.o
+COMMONOBJS = tun.o dns.o read.o encoding.o login.o base32.o md5.o common.o
 CLIENTOBJS = iodine.o
 CLIENT = ../bin/iodine
 SERVEROBJS = iodined.o user.o
diff -urNad iodine~/tests/Makefile iodine/tests/Makefile
--- iodine~/tests/Makefile	2008-03-01 18:52:44.000000000 +0100
+++ iodine/tests/Makefile	2008-03-01 18:53:37.000000000 +0100
@@ -1,7 +1,7 @@
 CC = gcc
 TEST = test
-OBJS = test.o base32.o base64.o read.o dns.o encoding.o login.o user.o 
-SRCOBJS = ../src/base32.o  ../src/base64.o ../src/read.o ../src/dns.o ../src/encoding.o ../src/login.o ../src/md5.o ../src/user.o
+OBJS = test.o base32.o read.o dns.o encoding.o login.o user.o 
+SRCOBJS = ../src/base32.o ../src/read.o ../src/dns.o ../src/encoding.o ../src/login.o ../src/md5.o ../src/user.o
 
 OS = `uname | tr "a-z" "A-Z"`
 
diff -urNad iodine~/tests/test.c iodine/tests/test.c
--- iodine~/tests/test.c	2007-12-01 01:43:06.000000000 +0100
+++ iodine/tests/test.c	2008-03-01 18:53:46.000000000 +0100
@@ -49,9 +49,6 @@
 	test = test_base32_create_tests();
 	suite_add_tcase(iodine, test);
 
-	test = test_base64_create_tests();
-	suite_add_tcase(iodine, test);
-
 	test = test_dns_create_tests();
 	suite_add_tcase(iodine, test);
 
